StellarDotnetSdk/LedgerKeys/LedgerKeyContractCode.cs

```diff
@@ -6,11 +6,11 @@ namespace StellarDotnetSdk.LedgerKeys;
 public class LedgerKeyContractCode : LedgerKey
 {
     /// <summary>
-    ///     Constructs a <c>LedgerKeyContractCode</c> object from a base-64 encoded string of the hash of the ledger entry.
+    ///     Constructs a <c>LedgerKeyContractCode</c> object from a hex encoded string of the hash of the ledger entry.
     ///     Use this to fetch contract wasm byte-code.
     /// </summary>
-    /// <param name="base64String">A base-64 encoded string.</param>
-    public LedgerKeyContractCode(string base64String) : this(Convert.FromBase64String(base64String))
+    /// <param name="hexString">A hex encoded string.</param>
+    public LedgerKeyContractCode(string hexString) : this(Convert.FromHexString(hexString))
```

Comment:
Where did you find that this should be a hex string? In the Java SDK this is just a base 64 just like you have it here.

Reply:
Well, it depends on how the user (or we) would view it. For reference:
- Soroban CLI: Hex
- Stellar Flutter SDK: Hex
- Laboratory's XDR Viewer: Base64
- Java SDK: Base64
I preferred hex because I thought Soroban CLI and the Flutter SDK could be considered "more official".

Reply:
Good reasoning behind this change. Are there not more locations where we should change this? would only like to have a more descriptive documentation. `A hex-encoded string of what?`

Reply:
Good reasoning behind this change but it seems that Stellar Flutter SDK has `A hash string of the Wasm bytes of a compiled smart contract.` if I read this code. Right?:

```dart
XdrLedgerKeyContractCode(XdrHash(Util.hexToBytes(wasmId)),
    XdrContractEntryBodyType.DATA_ENTRY);
```

Reply:
No, XdrHash is actually just a wrapper object containing the byte array, not a string as it seems, just like the generated Xdr.Hash class in our SDK.

Reply:
I've renamed the parameter to wasmHash to be more inline with other places. Also updated the description to match with the others.